stdlib/Dates/docs/src/index.md

```diff
@@ -427,7 +427,7 @@ julia> collect(dr)
 
 ## Adjuster Functions
 
-As convenient as date-period arithmetics are, often the kinds of calculations needed on dates
+As convenient as date-period arithmetic are, often the kinds of calculations needed on dates
```

Comment:
"arithmetic is" ?

Reply:
Reverted in https://github.com/JuliaLang/julia/pull/27976/commits/c551994938c7ffaebf79113dca685be26c190951